<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>How to use results of genetic epidemiological analyses to improve our understanding of biology?</title>
    <meta charset="utf-8" />
    <meta name="author" content="Julia Romanowska" />
    <script src="libs/clipboard/clipboard.min.js"></script>
    <link href="libs/xaringanExtra-clipboard/xaringanExtra-clipboard.css" rel="stylesheet" />
    <script src="libs/xaringanExtra-clipboard/xaringanExtra-clipboard.js"></script>
    <script>window.xaringanExtraClipboard(null, {"button":"Copy Code","success":"Copied!","error":"Press Ctrl+C to Copy"})</script>
    <script src="libs/xaringanExtra_fit-screen/fit-screen.js"></script>
    <link rel="stylesheet" href="xaringan-themer.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">
class: center, middle, inverse, title-slide

# How to use results of genetic epidemiological analyses to improve our understanding of biology?
### Julia Romanowska
### Dec.Â 4th, 2020

---




class: left, inverse, bottom

# OVERVIEW

## What kind of results?
## How to expand the results set?
## Genetic results
## Epigenetic results
## Heterogenous results

---

## What kind of results?

- RR, RRR
- meQTLs = SNP + CpG
- DMR / VMR
- CpGs / regulatory regions / SNPs / genes

    - gene body, promoter, enhancer,
    - CGI, open sea, CGI shore, CGI shelf

&lt;br&gt;&lt;br&gt;&lt;br&gt;
&lt;p style="font-size: small; font-style: italic; text-align: left;"&gt;TIP: to view my notes, press P&lt;/p&gt;

???

- relative risk (RR):    
*genetic effect*
- relative risk ratio (RRR):    
*interaction effect*
- methylation quantitative loci (meQTLs):    
*SNP-CpG pair, where the methylation level on the CpG is dependent on the allele at the SNP locus*
- differentially / variably methylated regions (DMR / VMR):    
*set of CpGs where the methylation either distinguishes between phenotype groups (DMR) or has highest variability among a group (VMR)*
&lt;br&gt;
- taken together, we get many *CpGs* and *SNPs*, which can be assigned to *genes* or *regulatory regions*, e.g.:

    - gene body, gene promoter (usually right next to their gene), gene enhancers (can be quite far away)
    - CpG-islands (CGI): *CpG-dense regions*
    - CGI-shore: *up to 2kb from CGI, usually hypomethylated*
    - CGI-shelf: *between 2kb and 4kb of CGI*
    - open sea: *rest of the genome*

---

## How to expand the results set?

- search engines:

    - Omics Discovery Index (https://www.omicsdi.org/)
    - GWAS catalog (https://www.ebi.ac.uk/gwas/) by EMBL-EBI
    - EWAS atlas (http://bigd.big.ac.cn/ewas/)
    - EWAS catalog (http://www.ewascatalog.org/) by Univ.of Bristol, UK
    - mQTL db (http://www.mqtldb.org/)
    - OpenTarget (https://www.targetvalidation.org/)

???

Sometimes, we would like to check our method on another dataset - there are more and more datasets freely available, while for some we need to apply to gain access. Some still are not available, but then we can usually find results from a certain analysis ran on them. All these we can search for through various search engines. Each of these mentioned resources has a different focus, but they usually give overlapping results, so it's useful to try several.

---

class: left, inverse, bottom

## How to interpret the result set?

### genetic data (SNPs, genes, proteins)
### DNA methylation data (CpGs, regions)

???

Let's assume then that we found a decent set of SNPs or CpGs that are somehow significant for our research question. We note them down in the Results section of our manuscript, but then comes the Discussion section - what to write there? How to interpret the results? I will show you several possibilities, first when focusing on the genetics only, and then including also epigenetics.

---

## Genetic results

- **SNPs / CpGs / regions**

    - visualize in ensembl (http://www.ensembl.org)

- **genes =&gt; proteins**

    - protein-protein interactions: String DB (https://string-db.org/)
    - information about proteins: UniProt (https://www.uniprot.org/)

- **genes =&gt; GO-terms, biological pathways**

    - enrichment: gProfiler (online https://biit.cs.ut.ee/gprofiler/gost)    
    ExpressionAtlas (https://www.ebi.ac.uk/gxa/genesetenrichment)    
    clusterProfiler (R package: https://guangchuangyu.github.io/software/clusterProfiler/)
    - pathways: ConsensuPathDB (http://consensuspathdb.org/)    
    Reactome (https://reactome.org/)

- **SNPs/genes =&gt; GWAS catalogue or OpenTarget**

    - what are they already associated with?

???

First of all, we can visualize our set of genomic loci - perhaps they cluster somehow? This can be done in ensembl.

If we're interested in proteins that these genes code, we can check whether the genes form an interaction network and check information about specific proteins.

If we want to understand what these genes have in common, perhaps we might check enrichment in gene-ontology terms or biological pathways. This can show whether the majority of the genes are involved in a certain biological process and whether it's significant, i.e., it's less probable that we end up in the same biological process when just choosing genes randomly.

We might also check what is already known about this SNP or gene.

---

## Epigenetic results

- **regulatory regions**

    - find a promoter or enhancer of a gene and vice-versa
    - GenEhancer (GeneCards https://www.genecards.org/)&lt;sup&gt;1&lt;/sup&gt;
    - JasparDB (http://jaspar.genereg.net/)

- **CpGs within promoters**

    - find what TFs bind there^[visualize JasparDB information in ensembl: http://jaspar.genereg.net/genome-tracks/#ensembl_tracks]

.footnote[&lt;sup&gt;1&lt;/sup&gt; Ref: https://doi.org/10.1093/database/bax028]

???

When it comes to the epigenetic data analysis, one can still try to find a set of genes to go along with interpretation, e.g., by taking the genes nearest the significant CpGs. However, as we know, genes are actually not occupying so much of the whole genome. Moreover, when doing _epi_genetic analysis, we would like to focus on the _epi_genetic processes and these start in the non-gene regions. Therefore, it's best to search for some information about the regulatory regions where the CpGs are located.

There is more and more tools that focus on the regulatory regions. We can try pairing promoters or enhancers with genes they regulate, or try to find which *transcription factors* (TFs) bind in vicinity of a CpG located within a promoter region.

---

## Epigenetic results

- **find enrichment with a special tool**

    - very good tutorial: https://tinyurl.com/y69wn7sv
    - [RegioneR R package](https://bioconductor.org/packages/release/bioc/html/regioneR.html)&lt;sup&gt;1&lt;/sup&gt;
    - [MethylGSEA R package](https://bioconductor.org/packages/release/bioc/html/methylGSA.html)&lt;sup&gt;2&lt;/sup&gt;

.footnote[&lt;sup&gt;1&lt;/sup&gt; Ref: https://doi.org/10.1093/bioinformatics/btv562;    
&lt;sup&gt;2&lt;/sup&gt; Ref: https://doi.org/10.1093/bioinformatics/bty892]

???

When we have many CpGs, we could check for their enrichment in being part of a certain regulatory region or regulatory process.

---

## BONUS

- Hetionet

https://het.io/
    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"highlightStyle": "solarized-light",
"highlightLines": true,
"countIncrementalSlides": false
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
